src/bin/pg_autoctl/file_utils.c

```diff
@@ -622,59 +592,20 @@ search_path(const char *filename, char ***result)
 			*sep = '\0';
 		}
 
-		join_path_components(candidate, path, filename);
-		canonicalize_path(candidate);
+		strlcpy(result->entries[result->total++], path, MAXPGPATH);
```

Comment:
This line can be removed afaict. (see other comment too)

Reply:
Removed.